lib/web/views/results/_table.html

```diff
@@ -0,0 +1,26 @@
+{{ if eq (len .Results) 0 }}
+  {{ template "results/_no_result" }}
+{{ else }}
+  <table class="table table-result">
```

Comment:
Are these classes mixing with multiple classes from another framework?
it seems that we have `table`, `table-result`, `table-data` and `table-header` here.
Should we make it properly with the same class module?

Reply:
Not sure why I added these class back then but removed it in [20e0722](https://github.com/carryall/go-google-scraper-challenge/pull/128/commits/20e0722980a16735c51e0269d03a9c5301b38dac) and use the tag instead to style the base table 😅